### DISCUSSION TITLE
CANCELLED September Meetup

### DISCUSSION THREAD

**Discussion by frenchy64** [madclj/madclj.com#18] [Announcements]
👍 to RSVP

https://madclj.com/

Wednesday, September 10th, 2025
6:30 PM to 9:00 PM CT

[StartingBlock Madison](https://www.google.com/maps/search/?api=1&query=43.081245%2C%20-89.374054)
821 E Washington Ave 2nd floor · Madison, WI

TBD

**Comment by frenchy64** [comment]
@dpetranek could you present something this week?

Other ideas:
- browse the Clojure/conj talks & discuss
- browse latest Clojure 1.12.2 features
- port [Celeste](https://www.lexaloffle.com/bbs/?tid=2145) to [Fennel](https://fennel-lang.org/)
- explore reproducible Clojure uberjar: https://github.com/frenchy64/reproducible-central/pull/1
- enhance tools.build to produce reproducible artifacts https://github.com/frenchy64/tools.build/pull/1
  - ditto leiningen
- explore json-schema -> malli translator

**Reply by dpetranek** [reply]
I don't have anything prepared right now, I've been wanting to explore core.async.flow, but I've been holding off until the virtual threads edition is ready. I think Celeste to Fennel could be fun!

**Comment by frenchy64** [comment]
I won't be able to attend so I need to cancel today's meetup.
